src/main/java/com/cloudogu/smeagol/wiki/domain/Author.java

```diff
@@ -0,0 +1,52 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.Preconditions;
+
+import java.util.Objects;
+
+/**
+ * Author value object represents the author of a wiki page. An author consists of an display name and a email address.
+ */
+public class Author {
+
+    private DisplayName displayName;
+    private Email email;
+
+    public Author(DisplayName displayName, Email email) {
+        this.displayName = Preconditions.checkNotNull(displayName);
+        this.email = Preconditions.checkNotNull(email);
+    }
+
+    public DisplayName getDisplayName() {
+        return displayName;
+    }
+
+    public Email getEmail() {
+        return email;
+    }
+
+    public String asString() {
+        StringBuilder builder = new StringBuilder(displayName.getValue());
+        builder.append(" <").append(email.getValue()).append(">");
+        return builder.toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)